test/keys.js

```diff
@@ -13,6 +15,19 @@ describe('keys', function() {
     eq(R.keys(obj).sort(), ['a', 'b', 'c', 'd', 'e', 'f']);
   });
 
+  it('should work when Object.keys is not present', function() {
+    var code =
+      '(function(require) { ' +
+      'var R = require(".."); ' +
+      'var eq = require("./shared/eq"); ' +
+      '' +
+      'Object.keys = void 0; ' +
```

Comment:
This should come before requiring Ramda, I believe, as Ramda may already have saved a reference to `Object.keys` by this point. Also, I think `delete Object.keys` would be slightly better here, as it would more accurately reflect the fact that the `keys` property is absent in old browsers.